### DISCUSSION TITLE
Help with themes and LSP

### DISCUSSION THREAD

**Discussion by HarrisonHemstreet** [helix-editor/helix#4621] [Q&A]
From what I've seen from YouTubers, it looks like their helix config had rust analyzer all set up and a long list of themes out of the box. But when I install helix, I'm not getting any of that. Was there a recent update that removed a lot of the default stuff from helix such as preinstalled LSPs and themes?

If this is the case, will someone please point me to a good tutorial on how I can install those themes I'm missing and get my LSP set up?

**Comment by the-mikedavis** [comment]
If you are missing the default themes, you most likely missed the step for linking the runtime directory: https://github.com/helix-editor/helix/blob/48a3965ab43718ce2a49724cbcc294b04c328b81/README.md?plain=1#L46-L53

Rust-analyzer and other language servers should work out of the box but they need to be installed https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#rust_analyzer
